<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLCDNumber" name="lcdnumberTime">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonExit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/exit.png</normaloff>:/image/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>168</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Choice">
     <item>
      <widget class="QRadioButton" name="radioButtonFirst">
       <property name="text">
        <string>人机先手</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSecond">
       <property name="text">
        <string>人机后手</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Play">
     <item>
      <widget class="QToolButton" name="toolButtonGamer">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/playWithGamer.png</normaloff>:/image/playWithGamer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonCompuer">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/playWithComputer.png</normaloff>:/image/playWithComputer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>111</width>
      <height>208</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Tern">
     <item>
      <widget class="QLabel" name="labelBalck">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/image/black.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWhite">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/image/white.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
